### DISCUSSION TITLE
PWA with idom

### DISCUSSION THREAD

**Discussion by seeclearvision** [idom-team/idom#655] [Question] [answered]
Is there any plan on creating a Progressive Web App using idom?

**Comment by Archmonger** [comment, marked as answer]
PWAs are possible. IDOM is a user interface framework and actually has no direct relation to to making a website Progressive Web Application (PWA) compatible. Any website can become a PWA if [the developer creates a `manifest.json`](https://pwa-workshop.js.org/1-manifest/#using-a-manifest-generator).

But assuming you actually meant to say Single Page Application (SPA)...
Right now SPAs are possible, albeit without any form of URL routing (which is planned https://github.com/idom-team/idom/issues/569).
